.nais/dev.json

```diff
@@ -13,7 +13,7 @@
     "TOGGLE_SEND_AKTIVITETSKRAV_VARSLER": "true",
     "TOGGLE_INFOTRYGD_KAFKA_CONSUMER": "true",
     "TOGGLE_UTBETALING_KAFKA_CONSUMER": "true",
-    "TOGGLE_MICROFRONTEND_ENABLING": "false",
+    "TOGGLE_MICROFRONTEND_ENABLING": "true",
```

Comment:
Bør vi vente med denne til alt er klart?

Reply:
@AudunSorheim Ja, dette er for `dev.json`. Kanskje litt forvirrende, siden alt pleier å være på  by default der.